source/change-streams/change-streams.rst

```diff
@@ -98,7 +98,7 @@ If the aggregation with ``$changeStream`` specified completes successfully, the
     /**
      * Describes the type of operation represented in this change notification.
      */
-    operationType: "insert" | "update" | "replace" | "delete" | "invalidate" | "drop" | "dropDatabase";
+    operationType: "insert" | "update" | "replace" | "delete" | "invalidate" | "drop" | "dropDatabase" | "rename";
```

Comment:
Drive-by fix, "rename" is another operationType that can be returned by a change stream.